src/components/ColorPicker/__tests__/ColorPicker.spec.ts

```diff
@@ -101,6 +101,37 @@ describe('ColorPicker', () => {
     });
   });
 
+  it('should allow pasting a hex value in the hex input', async () => {
+    const { getByTestId } = renderComponent({
+      props: initialProps
+    });
+
+    const colorPickerInput = getByTestId('color-picker-input');
+    expect(colorPickerInput).toBeInTheDocument();
+
+    await userEvent.click(colorPickerInput);
+
+    let colorPickerHexInput: HTMLInputElement | null = null;
+
+    await waitFor(() => {
+      colorPickerHexInput = screen.getByTestId(
+        'color-picker-hex-input'
+      ) as HTMLInputElement;
+      expect(colorPickerHexInput).toBeInTheDocument();
+      expect(colorPickerHexInput).toBeVisible();
+    });
```

Comment:
This is another way to do this too, I think its a bit quicker since you don't need to wrap w/ the `await waitFor`. Not saying you have to change it but wanted to note in case you hadn't seen it done this way. You just have to add `findByTestId` on line 105
```suggestion
    let colorPickerHexInput = await findByTestId('color-picker-hex-input') as HTMLInputElement;

    expect(colorPickerHexInput).toBeVisible();
```

Reply:
Yep, I've used the find* queries too.  I agree that approach is more concise, I'll update.